### ISSUE TITLE
Publish new version that is compatible with LiveView 1.0?

### ISSUE THREAD

**Issue by Munksgaard** [maxmarcon/live_select#88]
LiveView 1.0 has just been released. I'd love to upgrade, but live_select requires `phoenix_live_view ~> 0.19`. I know that I can use `override: true` in my mix.deps, it would be nicer to release a new version of live_select that was fully compatible.

Related to #85.

Edit: And by the way, thank you for your work on this component. It is great!

**Comment by maxmarcon** [comment]
Thanks for the heads-up. I just updated to LV 1.0.0 and tried to compile, but I get this error:

```
Generated phoenix_live_reload app
==> phoenix_ecto
Compiling 7 files (.ex)
Generated phoenix_ecto app
==> live_select
Compiling 11 files (.ex)
error: undefined variable "my_form"
  lib/support/live_select_web/live/showcase_live.ex:244: LiveSelectWeb.ShowcaseLive.Render.live_select/1


== Compilation error in file lib/support/live_select_web/live/showcase_live.ex ==
** (CompileError) lib/support/live_select_web/live/showcase_live.ex: cannot compile module LiveSelectWeb.ShowcaseLive.Render (errors have been logged)
```

no time to investigate right now. If you have a clue, I'm all ears!

Thanks

**Comment by maxmarcon** [comment]
I just published a new version 1.4.4 compatible with LV 1.0.0
